src/nginxconfig/generators/conf/website.conf.js

```diff
@@ -60,13 +60,24 @@ const httpsListen = domain => {
     const config = [];
 
     // HTTPS
-    config.push(['listen', `${domain.server.listenIpv4.computed === '*' ? '' : `${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' : ''}`]);
+    config.push(['listen', `${domain.server.listenIpv4.computed === '*' ? '' :`${domain.server.listenIpv4.computed}:`}443 ssl${domain.https.http2.computed ? ' http2' :''}`]);
```

Comment:
You still have a diff here when nothing should be changing with this line, please revert :)

Reply:
Whoops,  sorry for that!